zone/entity.cpp

```diff
@@ -1611,7 +1611,7 @@ void EntityList::QueueClientsByTarget(Mob *sender, const EQApplicationPacket *ap
 
 		TargetsTarget = Target->GetTarget();
 
-		bool Send = false;
```

Comment:
Should we just use the parameter at this point?

Reply:
I don't think so.  It's kind of a crappy overload but it felt better than rewriting the whole thing.  When we send a packet to clear the window, we always send it.  Hence, if clear is set so is send..  But I think it's clearer as written.  Let me know.